src/firebase-db.test.ts

```diff
@@ -88,7 +88,7 @@ describe("Firestore", () => {
       resourceUrl: "http://example/resource",
       toolId: "activity-player.concord.org",
       userType: "learner",
-      runRemoteEndpoint: ""
+      runRemoteEndpoint: "https://portal.com/learner/1234"
```

Comment:
Should we use `example.com` here in case the tests ever accidentally use this endpoint?

Reply:
done